.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+### Description
+
+<!-- Required -->
+<!-- Description of the issue] -->
+
+### Steps to Reproduce
+
+<!-- Required -->
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+**Expected behavior:**
+
+<!-- What you expect to happen-->
+
+**Actual behavior:**
+
+<!-- What actually happens -->
+
+**Reproduces how often:**
+<!-- What percentage of the time does it reproduce?-->
```

Comment:
`<!-- How often does the issue happen, on a scale from 1 (rare) to 5 (always)? -->`